src/content/docs/crowdin/project-management/sources/version-management.mdx

```diff
@@ -28,35 +26,130 @@ Example of branch structure in the version control system (VCS) and Crowdin:
 
-* Integrate Crowdin projects with [GitLab](/gitlab-integration/), [GitHub](/github-integration/), [Bitbucket](/bitbucket-integration/), or [Azure Repos](/azure-repos-integration/). It’s a recommended option, as branches selected for localization in your version control system (VCS) will be created in Crowdin automatically.
-* Use [CLI](https://crowdin.github.io/crowdin-cli/) or [API](/developer/api/) to add new branches.
-* Create branches [manually](/file-management/#branches-and-folders).
+### Creating Branches
 
-<Aside type="caution">
-  The first files you upload to the Crowdin project must contain all the original strings and should be placed in the corresponding branch. For all article examples, a branch with original texts is named *Master*.
-</Aside>
+You can create branches manually using the web interface (UI), API, or CLI, or automatically by connecting your project to a version control system. The method you choose may depend on your project type or integration preferences.
+
+#### Creating Branches via VCS Integration
+
+Integrate Crowdin projects with [GitLab](/gitlab-integration/), [GitHub](/github-integration/), [Bitbucket](/bitbucket-integration/), or [Azure Repos](/azure-repos-integration/). When you select branches for localization in your VCS, they’ll be created automatically in Crowdin.
+
+#### Creating Branches via Web Interface
+
+To create branches via the web interface (UI), follow these steps:
+
+<Tabs syncKey="projectType">
+  <TabItem label="File-based project">
+    <Steps>
+      1. Open your project and go to **Sources > Files**.
+      2. Click **New Version Branch**.
+      3. In the appeared dialog, enter the branch name.
+      4. In the **Duplicate Strings** section, select how duplicated strings should be handled.
+      5. Click **Create** to finish.
+    </Steps>
+  </TabItem>
+  <TabItem label="String-based project">
+    <Steps>
+      1. Open your project and go to the **Branches** tab.
+      2. Click **New Version Branch**.
+      3. In the appeared dialog, enter the branch name.
+      4. Select **Create empty branch** or **Clone existing branch**.
+      5. If cloning, select the source branch from the dropdown that appears.
+      6. Click **Create** to finish.
+    </Steps>
+  </TabItem>
+</Tabs>
 
 #### Branch Creation using API
 
-To create a branch via API, use the [Add Branch](/developer/api/v2/#operation/api.projects.branches.post) method.
+You can use the Crowdin API to programmatically create branches. The available method depends on the project type:
+
+<Tabs syncKey="projectType">
+  <TabItem label="File-based project">
+    * [Add Branch](https://support.crowdin.com/developer/api/v2/#tag/Source-Files/operation/api.projects.branches.post)
+  </TabItem>
+  <TabItem label="String-based project">
+    * [Add Branch](https://support.crowdin.com/developer/api/v2/string-based/#tag/Branches/operation/api.projects.branches.post)
+  </TabItem>
+</Tabs>
```

Comment:
The Tabs component for only one link is too much, just put two links in a simple list